apps/remix-dapp/src/locales/en/udapp.json

```diff
@@ -61,13 +61,13 @@
   "udapp.reset": "Reset",
   "udapp.delete": "Delete",
   "udapp.injectedTitle": "Unfortunately it's not possible to create an account using injected provider. Please create the account directly from your provider (i.e metamask or other of the same type).",
-  "udapp.createSmartAccount": "Create Safe Smart Account",
-  "udapp.createSmartAccountAlpha": "Create Safe Smart Account [ALPHA]",
+  "udapp.createSmartAccount": "Create Gnosis Safe Smart Account",
+  "udapp.createSmartAccountAlpha": "Create Gnosis Safe Smart Account [ALPHA]",
   "udapp.createSmartAccountDesc1":"ERC-4337 introduces smart contracts accounts, which are more advanced than traditional EOAs. These accounts offer features like MULTISIGNATURE for extra security, GASLESS transactions and custom transaction rules for added flexibilitity.",
-  "udapp.createSmartAccountDesc2":"These smart contract accounts are usually referred as SMART ACCOUNTS. A Safe Smart Account is a Smart Account with multi-signature functionality at its core.",
-  "udapp.createSmartAccountDesc3":"Smart Account will be created with the below account as OWNER. Owner account is used to sign each transaction(user-operation) made using Smart Account.",
-  "udapp.createSmartAccountDesc4":"Smart Account will by default have a PAYMASTER attached to it to execute GASLESS transactions, only for SEPOLIA testnet. ",
-  "udapp.createSmartAccountDesc5":"Smart Account creation will require signing of an initial transaction at next step.",
+  "udapp.createSmartAccountDesc2":"These smart contract accounts are usually referred as SMART ACCOUNTS. A Safe Smart Account is a Smart Account with multi-signature functionality of Gnosis Safe at its core.",
```

Comment:
These smart contract accounts are often referred to as Smart Accounts. A Safe Smart Account is a Smart Account with the multi-signature functionality of Gnosis Safe at its core.